concent_api/core/views.py

```diff
@@ -55,8 +55,22 @@ def send(request, message):
 
 @api_view
 @require_POST
-def receive(_request):
-    return "message sent"
+def receive(request, message):
```

Comment:
pylint returns warnings for this line.

```
W: 80,12: Unused argument 'request' (unused-argument)
W: 80,21: Unused argument 'message' (unused-argument)
```

Please always make sure that there are no unfixed warnings before submitting a pull request.